src/templates/admin/index.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'admin/master.html' %}
+{% block body %}
+{{ super() }}
+<div class="container">
+    <div class="row">
+        <div class="col-sm-10 col-sm-offset-1">
+            <h1>Admin home</h1>
+            <p class="lead">
+                Authentication
+            </p>
+            {% if current_user.is_authenticated %}
+            <p>
+                You are currently logged in as {{ current_user.email }}, with role {{ current_user.role }}
+            </p>
+            {% endif %}
+            {% if not current_user.is_authenticated %}
```

Comment:
otherwise it would not work if you access the admin page, while not having an account. I also wanted to have a different message when you do have an account, but are not admin
1. admin
2. no account
3. account + no admin